docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1584,9 +1584,38 @@ public class PostGatewayFilterFactory extends AbstractGatewayFilterFactory<PostG
 
 === Writing Custom Global Filters
 
-TODO: document writing Custom Global Filters
+In order to write a custom global filter, you will need to implement `GlobalFilter` interface. This will apply the filter to all requests. (This interface and its usages are subject to change in future milestones)
```

Comment:
"implement the `GlobalFilter` interface"

Reply:
You got it! Should that sentence also be removed from the other section in the documentation?

https://cloud.spring.io/spring-cloud-gateway/single/spring-cloud-gateway.html#_global_filters